File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroSearchPatientInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroSearchPatientInfo
    {
        private string _birth;
        private string _patientName1;
        private string _patientName2;

        public NuroSearchPatientInfo()
        {
            
        }

        public NuroSearchPatientInfo(string birth, string patientName1, string patientName2)
        {
            _birth = birth;
            _patientName1 = patientName1;
            _patientName2 = patientName2;
        }

        public string Birth
        {
            get { return _birth; }
            set { _birth = value; }
        }

        public string PatientName1
        {
            get { return _patientName1; }
            set { _patientName1 = value; }
        }

        public string PatientName2
        {
            get { return _patientName2; }
            set { _patientName2 = value; }
        }
    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroSearchPatientInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroSearchPatientInfo {
	private String patientCode;
	private String patientName1;
	private String patientName2;
	private String birth;

	public NuroSearchPatientInfo(String patientCode, String patientName1,
			String patientName2, String birth) {
		super();
		this.patientCode = patientCode;
		this.patientName1 = patientName1;
		this.patientName2 = patientName2;
		this.birth = birth;
	}

	public String getPatientCode() {
		return patientCode;
	}

	public void setPatientCode(String patientCode) {
		this.patientCode = patientCode;
	}

	public String getPatientName1() {
		return patientName1;
	}

	public void setPatientName1(String patientName1) {
		this.patientName1 = patientName1;
	}

	public String getPatientName2() {
		return patientName2;
	}

	public void setPatientName2(String patientName2) {
		this.patientName2 = patientName2;
	}

	public String getBirth() {
		return birth;
	}

	public void setBirth(String birth) {
		this.birth = birth;
	}

}
